sorc/build_gfs_bufrsnd.sh

```diff
@@ -5,6 +5,7 @@ source ./machine-setup.sh > /dev/null 2>&1
 cwd=`pwd`
 
 USE_PREINST_LIBS=${USE_PREINST_LIBS:-"true"}
+source ../versions/build.ver
```

Comment:
Source machine-setup.sh here instead, the machine-setup.sh script source build.ver for you. See other build scripts for examples.

Reply:
Changed it. Thanks.